pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -70,23 +73,23 @@ export default {
   },
 
   computed: {
-    onInput() {
+    onInput(): any | void {
```

Comment:
I've struggled to come up with a good type for this computed prop.. I'm open to suggestions.. Otherwise, we might have to live with the `typescript-eslint/no-explicit-any` warning until we come up with something that appropriately fits.

Reply:
My IDE suggests it quite easily the 2 returned values which you can put as union type like this `void | DebouncedFunc<() => void>`.
Not sure if you have the suggestion as well or not 🤔

Reply:
Thanks for the suggestion, I like this approach much better 🙂